apps/moderation/fields.py

```diff
@@ -93,12 +94,13 @@ def serialize_instance(instance: Model) -> SerializedModel:
         if is_string_field and getattr(instance, field.name, None) is None:
             setattr(instance, field.name, '')
     value_set = [instance]
-    if instance._meta.parents:
-        value_set += [
-            getattr(instance, f.name)
-            for f in list(instance._meta.parents.values())
-            if f is not None
-        ]
+    # patch: disable adding parent models because patched PythonSerializer includes parent fields
```

Comment:
@alashow can you please explain this in more detail? Was something broken?

Reply:
https://discord.com/channels/799456464120971294/882576239515279380/895601531930435585